src/components/pages/TrophyRoom/RenderTrophyRoom.js

```diff
@@ -44,19 +49,22 @@ const RenderTrophyRoom = props => {
   return (
     <>
       <Header displayMenu={true} title="Trophy Room" />
+      <Button style={{ margin: '1rem' }} onClick={dashboard}>
+        Back to Child Dashboard
+      </Button>
       <div className="trophy-container">
         <Leaderboard child={props.child} />
         <div className="custom-divider"></div>
         <div className="modal-container">
           <h2 className="h2" onClick={showAchievementsModal}>
             Achievements
           </h2>
-          <Modal
+          <div
```

Comment:
I think the only issue with having this not be a modal would be that if in the future more achievements are added then there won't be enough screen-size to hold them without scrolling (or that they would extend past the leaderboard which could look uneven/awkward), since we're sort of approaching the full length with the current achievement list.  Also, do you prefer having the achievements displayed this way in list-form as opposed to being help in a popup?  I'm trying to decide what I think of it

Reply:
I agree, the page would look off when a user gets more achievements. I liked the popup more but David didn't think that was ideal for the UI. Maybe we should just leave it as a popup for now.

Reply:
I agree with you, perhaps the idea of having too many achievements is something he didn't consider.

Reply:
Changed it back to a pop-up and pushed to the branch.